apis/server/image_bridge.go

```diff
@@ -117,3 +117,20 @@ func (s *Server) removeImage(ctx context.Context, rw http.ResponseWriter, req *h
 	rw.WriteHeader(http.StatusNoContent)
 	return nil
 }
+
+// postImageTag adds tag for the existing image.
+func (s *Server) postImageTag(ctx context.Context, rw http.ResponseWriter, req *http.Request) error {
+	name := mux.Vars(req)["name"]
+
+	targetRef := req.FormValue("repo")
+	if tag := req.FormValue("tag"); tag != "" {
```

Comment:
if tag is "", do we need set default value `latest`?

Reply:
I have handled it in the deamon so that we can keep the bridge simple.